src/Servers/Kestrel/test/Interop.FunctionalTests/HttpClientHttp2InteropTests.cs

```diff
@@ -1416,7 +1416,6 @@ public async Task Settings_MaxHeaderListSize_Client(string scheme)
         // Settings_InitialWindowSize_Lower_Client - Not configurable.
 
         [Theory]
-        [QuarantinedTest]
```

Comment:
No failure or issue for this one, propose un-quarantining and monitoring for future failure.